app/src/interfaces/input-rich-text-html/useImage.ts

```diff
@@ -141,7 +141,7 @@ export default function useImage(
 		if (img === null) return;
 
 		const queries: Record<string, any> = {};
-		const newURL = new URL(img.imageUrl);
+		const newURL = new URL(img.imageUrl, 'file://');
```

Comment:
Looks like we're only using `URL` to set the search params. Maybe it's easier to create a new SearchParams object first, and then `imageUrl + searchParams.toString()` that? 🤔 

Feels a tad cleaner than having to strip out the file:// protocol later

Reply:
I tried. I was wrong.